src/server/frontend_wayland/xdg_shell_stable.h

```diff
@@ -20,6 +20,8 @@
 #include "xdg-shell_wrapper.h"
 #include "window_wl_surface_role.h"
 
+namespace mw = mir::wayland;
+
```

Comment:
Please don't introduce this alias to global namespace. In headers it is better to spell things in full